ivy/functional/backends/tensorflow/layers.py

```diff
@@ -452,9 +452,7 @@ def conv_general_transpose(
             axis=-1,
         )
     elif dims == 2:
-        res = tf.concat(
-            [
-                conv2d_transpose(
+        res = conv2d_transpose(
                     x[..., j : j + filters.shape[-2] // feature_group_count],
```

Comment:
Actually I see the test failing for the tensorflow backend in the CI checks. Not sure how you can modify this to work correctly.